ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantSessionManager.scala

```diff
@@ -12,7 +12,7 @@ import org.slf4j.LoggerFactory
 import scala.collection.concurrent.TrieMap
 import scala.concurrent.{ExecutionContext, Future}
 
-private[infrastructure] final class ParticipantSessionManager {
+private[testtool] final class ParticipantSessionManager {
```

Comment:
Because in order to have the `--list` argument print the test names, I needed to reference this class from `testtool.LedgerApiTestTool`. Definitely a hack and happy to do something else if you can point me in a better direction.